### ISSUE TITLE
Load_ndcs module as separate runnable module 

### ISSUE THREAD

**Issue by yevgenybulochnik** [pharmorg/OpenFDA#15]
We should consider making the load_ndcs module separately runnable from the web server itself. This would avoid the current process of commenting out part of the run.py module.

**Comment by finish06** [comment]
An approach to doing this is separate folder / code structure for the load_ndcs & connect_db modules.  Create a folder, e.g. `./build_data` and place the files in the such folder.  Then enable the load_ndcs module to run via command line and make a change to docker-compose so it is a separate container.  I think if we intend to decouple the code, we need to make the folder structure foretelling.  Thoughts?

**Comment by yevgenybulochnik** [comment]
makes sense, if we go the container route for loading data we likely need another `docker-compose.` maybe `docker-compose.build-data.yml`. This way we can specify (`docker-compose up -f docker-compose.build.yml`) when the data should be built. Otherwise on every docker-compose up it will run through the data. Another approach would be to build in some sort of check to see if the data has already been loaded.

**Comment by finish06** [comment]
`connect_db.py` is only used by `load_ndcs.py` therefore I think it can stay in the same directory as `load_ndcs.py`.  That may change in the future, but we can make changes when that happens.  I prefer the following:
```
build_data/
server/
```

**Comment by finish06** [comment]
I am struggling to import the helper function from the parent directory.  Thoughts?
